pkg/vadmin/install_packages_at.go

```diff
@@ -34,7 +35,8 @@ func (a *Admintools) InstallPackages(ctx context.Context, opts ...installpackage
 
 	status := genInstallPackageStatus(stdout)
 	if err != nil && len(status.Packages) == 0 {
-		_, logErr := a.logFailure("install_package", events.InstallPackagesFailed, stdout, err)
+		pkgErr := errors.New(err.Error() + "This may due to lack of memory resources.")
```

Comment:
Do we have a way to distinguish the cause of the error? Simply adding a string here can be misleading for future debugging.

Reply:
Thanks. Now it only logs a message if 0 package was installed.